analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataFactory.Package.cs

```diff
@@ -86,7 +85,7 @@ private void Install()
                 var versionArgument = Version == Constants.NuGetLatestVersion ? string.Empty : $"-Version {Version}";
                 var configFile = ValidatedNuGetConfigPath();
                 // Explicitly specify the NuGet config to use to avoid being impacted by the NuGet config on the machine running the tests
-                var args = $"install {Id} {versionArgument} -OutputDirectory {Path.GetFullPath(PackagesFolderRelativePath)} -NonInteractive -ForceEnglishOutput -ConfigFile {configFile}";
+                var args = $"install {Id} {versionArgument} -OutputDirectory \"{Environment.GetEnvironmentVariable("NUGET_PACKAGES")}\" -NonInteractive -ForceEnglishOutput -ConfigFile {configFile}";
```

Comment:
This should use `PackagesFolder` instead, shouldn't it? The fallback for caching would not work without the env variable.